omod/src/main/webapp/resources/scripts/navigator/navigator.js

```diff
@@ -25,74 +25,82 @@ var Navigator = {isReady: false}
 
 
 // TODO figure out out to set up mocking to test this
+// TODO this really isn't just the "keyboard controller" anymore as it adds buttons and supports mouse clickers
 function KeyboardController(formElement) {
 
     var initFormModels = function(formEl) {
         var formElement = formEl;
         if (!formElement) {
             formElement = $('div#content > form').first();
         }
+
         formElement.prepend('<ul id="formBreadcrumb" class="options"></ul>');
+        formElement.append('<div id="nav-buttons"><button id="prev-button" type="button" class="confirm" style="display:none"><icon class="fas fa-chevron-left"/></button><button id="next-button" class="confirm right" type="button"><icon class="fas fa-chevron-right"/></button></div>');
```

Comment:
Just inserts the two buttons in the form... see styling below that hides the buttons on large screens.

Reply:
```suggestion
        formElement.append(
          '<div id="nav-buttons">'
          + '<button id="prev-button" type="button" class="confirm" style="display:none">'
          + '  <icon class="fas fa-chevron-left"/>'
          + '</button>'
          + '<button id="next-button" class="confirm right" type="button">'
          + '  <icon class="fas fa-chevron-right"/>'
          + '</button>'
          + '</div>');
```